### DISCUSSION TITLE
OpenMCT vs Cosmos

### DISCUSSION THREAD

**Discussion by satzguyz** [nasa/openmct#5076] [Community Q&A] [answered]
Hello

Does the OpenMCT and Cosmos (https://cosmosc2.com/)  provides the same functionality ? Both seems open source. Just curious on advantage of one over other.

**Comment by ptcrash** [comment]
It looks like Cosmos is specialized for software service telemetry & ops whereas OpenMCT is a generalized platform that can be used any many other cases. The trade-off being OpenMCT isn't specialized for what you want and thusly requires more development to get it ingesting your data just the way you want it.

**Comment by akhenry** [comment, marked as answer]
@satzguyz Thanks for the question! Cosmos is a full telemetry and commanding system. Open MCT is a front-end that interfaces with telemetry and commanding software and provides additional functionality such as telemetry visualization, timeline displays, notebooks, operator status polling, and many other capabilities. 

Open source plugins are available to use Open MCT with [Yamcs](https://github.com/akhenry/openmct-yamcs) and [AIT](https://github.com/NASA-AMMOS/AIT-Core/tree/master/openmct). 

In principle it could be used with Cosmos as the backend, and if someone were to write that adapter we'd love for you to share that with the community as well.
